Mathlib/Topology/Algebra/Module/Star.lean

```diff
@@ -52,15 +54,18 @@ theorem continuous_decomposeProdAdjoint_symm [TopologicalAddGroup A] :
 #align continuous_decompose_prod_adjoint_symm continuous_decomposeProdAdjoint_symm
 
 /-- The self-adjoint part of an element of a star module, as a continuous linear map. -/
-@[simps]
+@[simps!]
 def selfAdjointPartL [ContinuousAdd A] [ContinuousStar A] [ContinuousConstSMul R A] :
     A →L[R] selfAdjoint A where
   toLinearMap := selfAdjointPart R
   cont := continuous_selfAdjointPart _ _
 #align self_adjoint_partL selfAdjointPartL
 
+-- Porting note: `simp only [selfAdjointPartL_toFun_coe]` proves this projection
+attribute [nolint simpNF] selfAdjointPartL_apply_coe
```

Comment:
This looks like `simps` is misconfigured, though it's fine that this PR was merged as is